_includes/getting-started.html

```diff
@@ -66,11 +66,13 @@ <h3>Brew User:</h3>
             <p>Install on Mac or Linux using Homebrew:</p>
 
             <div id="start-brew" class="highlight-code">
-              <p class="code-box" style="height: 60px; line-height: 30px;"><code style="position: absolute;"> $ brew install layer5io/mesheryctl
+              <p class="code-box" style="height: 85px; line-height: 30px;"><code style="position: absolute;"> $ brew tap layer5io/tap
```

Comment:
Increase the height to 95px to avoid the scrollbar.

Reply:
okay